generics/services/entity-management.js

```diff
@@ -25,7 +25,10 @@ const entityDocuments = function (filterData = 'all', projection = 'all') {
 	return new Promise(async (resolve, reject) => {
 		try {
 			// Function to find entity documents based on the given filter and projection
-			const url = entityManagementServiceUrl + CONSTANTS.endpoints.FIND_ENTITY_DOCUMENTS
+			const url =
+				interfaceServiceUrl +
+				process.env.ENTITY_MANAGEMENT_SERVICE_NAME +
```

Comment:
@Prajwal17Tunerlabs please update the variable names and test

Reply:
env variable names are updated .